nntrainer/graph/network_graph.cpp

```diff
@@ -857,4 +857,79 @@ std::vector<LayerNode> &NetworkGraph::getSorted() {
   return Sorted;
 }
 
+int NetworkGraph::initialize(std::shared_ptr<Manager> manager) {
```

Comment:
`NetworkGraph::iniaitialize` has been extracted from model for better testing & interface.

Reply:
This is just copy & paste.